libnetwork/store_linux_test.go

```diff
@@ -19,9 +21,14 @@ func TestBoltdbBackend(t *testing.T) {
 }
 
 func TestNoPersist(t *testing.T) {
-	testController, err := New(OptionBoltdbWithRandomDBFile(t))
+	configOption := func(c *config.Config) {
+		c.Scope.Client.Provider = "boltdb"
+		c.Scope.Client.Address = filepath.Join(t.TempDir(), "bolt.db")
+		c.Scope.Client.Config = &store.Config{Bucket: "testBackend"}
+	}
```

Comment:
[Each call to `t.TempDir()` returns a unique directory.](https://pkg.go.dev/testing#T.TempDir)
```suggestion
	dbfile := filepath.Join(t.TempDir(), "bolt.db")
	configOption := func(c *config.Config) {
		c.Scope.Client.Provider = "boltdb"
		c.Scope.Client.Address = dbfile
		c.Scope.Client.Config = &store.Config{Bucket: "testBackend"}
	}
```

Reply:
🤦 you're right. I thought to be smart, and wasn't

Reply:
Fixed 🙈 ❤️